modules/host/rpcsubscribe.go

```diff
@@ -193,6 +232,39 @@ func (h *Host) managedHandleUnsubscribeRequest(info *subscriptionInfo, pt *modul
 	return errors.AddContext(err, "failed to signal successfully unsubscribing from entries")
```

Comment:
This should also use `managedHandleFinalizeUnsubscribeRequest`

Reply:
done